### ISSUE TITLE
Please add if __name__ == "__main__":

### ISSUE THREAD

**Issue by erikbuck** [chr15m/PodSixNet#27]
I love PodSixNet and use it to teach inroductory network programming at a University. The ChatSever and ChatClient example is nearly perfect for my purposes. I have students subclass ChatSever and ChatClient, and the fact that code for creating ChatServer and ChatClient instances executes unconditionally in the example code causes problems.

Please place `if __name__ == "__main__":` around code that shouldn't execute when subclasses are created:

ChatClient.py
```
if __name__ == "__main__":
   if len(sys.argv) != 2:
       print("Usage:", sys.argv[0], "host:port")
       print("e.g.", sys.argv[0], "localhost:31425")
   else:
       host, port = sys.argv[1].split(":")
       c = Client(host, int(port))
       while 1:
           c.Loop()
           sleep(0.001)
```
-------------------------
ChatServer.py
```
if __name__ == "__main__":
   # get command line argument of server, port
   if len(sys.argv) != 2:
      print("Usage:", sys.argv[0], "host:port")
      print("e.g.", sys.argv[0], "localhost:31425")
   else:
      host, port = sys.argv[1].split(":")
      s = ChatServer(localaddr=(host, int(port)))
      s.Launch()

```

**Comment by chr15m** [comment]
@erikbuck this seems like a worthwhile change, thank you. Do you know how to make this modification and issue a PR? If you do so I will merge the change after testing. Thanks again.

**Comment by pauliyobo** [comment]
I could issue the PR if the issue hasn't been solved yet.

**Comment by chr15m** [comment]
Fixed in #29.
